ConsequencesTest/DepthHazardTest.cs

```diff
@@ -0,0 +1,52 @@
+﻿using USACE.HEC.Hazards;
+
+namespace ConsequencesTest;
+public class DepthHazardTest
```

Comment:
https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices

Reply:
I might change the names of these individual tests. We know from the class name already that these are tests on DepthHazard, so go ahead and remove the Test prefix from everyone. 

Stolen from Microsoft: When you're naming your tests, try to communicate these 3 things. 
    The name of the method being tested.
    The scenario under which it's being tested.
    The expected behavior when the scenario is invoked.